pkg/internal/apis/certmanager/validation/issuer.go

```diff
@@ -348,7 +348,7 @@ func ValidateACMEChallengeSolverDNS01(p *cmacme.ACMEChallengeSolverDNS01, fldPat
 				el = append(el, field.Required(fldPath.Child("rfc2136", "nameserver"), ""))
 			} else {
 				if _, err := util.ValidNameserver(p.RFC2136.Nameserver); err != nil {
-					el = append(el, field.Invalid(fldPath.Child("rfc2136", "nameserver"), "", "Nameserver invalid. Check the documentation for details."))
+					el = append(el, field.Invalid(fldPath.Child("rfc2136", "nameserver"), p.RFC2136.Nameserver, "Nameserver invalid. Check the documentation for details."))
```

Comment:
Could you update the message printed here to not guide people to `check the documentation for details` and instead provide some more actionable info? i.e. change to `must be a fully qualified domain name or IP address with port in the format host:port`?